media/js/base/sticky-promo.js

```diff
@@ -0,0 +1,16 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+(function() {
+    'use strict';
+
+    document.addEventListener('DOMContentLoaded', function() {
```

Comment:
The function was firing before the page was ready. If there's another more suitable document-ready function, lmk!

Reply:
Scripts should typically go at the end of the document body. The one here is being requested before the page content, making it render blocking and hence the need for the event listener here. I’d suggest moving the JS either into the page’s JS block, or alternatively into the existing page bundle.

Reply:
I moved the script below and it loaded without the need to wait. I'll move the entire block into the existing page bundle JS instead. Great callout!